pkg/epp/controller/inferencepool_reconciler_test.go

```diff
@@ -216,10 +216,11 @@ func TestXInferencePoolReconciler(t *testing.T) {
 		Version: v1alpha2.GroupVersion.Version,
 		Kind:    "InferencePool",
 	}
+	// TODO: change it to 8080
```

Comment:
Will change it back once https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/1330 gets merged